diff -Naur apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/AtlasHiveHookContext.java apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/AtlasHiveHookContext.java
--- apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/AtlasHiveHookContext.java	2019-05-06 12:55:33.000000000 -0500
+++ apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/AtlasHiveHookContext.java	2019-11-13 19:39:54.587194726 -0600
@@ -77,7 +77,7 @@
         this.knownObjects     = knownObjects;
         this.metastoreHook    = metastoreHook;
         this.metastoreEvent   = listenerEvent;
-        this.metastoreHandler = (listenerEvent != null) ? metastoreEvent.getIHMSHandler() : null;
+        this.metastoreHandler = (listenerEvent != null) ? metastoreEvent.getHandler() : null;
 
         init();
     }
@@ -248,10 +248,6 @@
                 case CREATEDATABASE:
                     databases.add(((CreateDatabaseEvent) metastoreEvent).getDatabase());
                     break;
-                case ALTERDATABASE:
-                    databases.add(((AlterDatabaseEvent) metastoreEvent).getOldDatabase());
-                    databases.add(((AlterDatabaseEvent) metastoreEvent).getNewDatabase());
-                    break;
                 case CREATETABLE:
                     tables.add(toTable(((CreateTableEvent) metastoreEvent).getTable()));
                     break;
@@ -289,4 +285,4 @@
     private static boolean isCreateAlterOperation(String operationName) {
         return operationName != null && operationName.startsWith(CREATE_OPERATION) || operationName.startsWith(ALTER_OPERATION);
     }
-}
\ No newline at end of file
+}
diff -Naur apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/AlterDatabase.java apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/AlterDatabase.java
--- apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/AlterDatabase.java	2019-05-03 00:22:00.000000000 -0500
+++ apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/AlterDatabase.java	1969-12-31 18:00:00.000000000 -0600
@@ -1,76 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- * <p/>
- * http://www.apache.org/licenses/LICENSE-2.0
- * <p/>
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.atlas.hive.hook.events;
-
-import org.apache.atlas.hive.hook.AtlasHiveHookContext;
-import org.apache.atlas.model.instance.AtlasEntity;
-import org.apache.atlas.model.instance.AtlasEntity.AtlasEntitiesWithExtInfo;
-import org.apache.atlas.model.notification.HookNotification;
-import org.apache.atlas.model.notification.HookNotification.EntityUpdateRequestV2;
-import org.apache.commons.collections.CollectionUtils;
-import org.apache.hadoop.hive.metastore.api.Database;
-import org.apache.hadoop.hive.metastore.events.AlterDatabaseEvent;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.Collections;
-import java.util.List;
-
-public class AlterDatabase extends CreateDatabase {
-    private static final Logger LOG = LoggerFactory.getLogger(AlterDatabase.class);
-
-    public AlterDatabase(AtlasHiveHookContext context) {
-        super(context);
-    }
-
-    @Override
-    public List<HookNotification> getNotificationMessages() throws Exception {
-        List<HookNotification>   ret      = null;
-        AtlasEntitiesWithExtInfo entities = context.isMetastoreHook() ? getHiveMetastoreEntities() : getHiveEntities();
-
-        if (entities != null && CollectionUtils.isNotEmpty(entities.getEntities())) {
-            ret = Collections.singletonList(new EntityUpdateRequestV2(getUserName(), entities));
-        }
-
-        return ret;
-    }
-
-    public AtlasEntitiesWithExtInfo getHiveMetastoreEntities() throws Exception {
-        AtlasEntitiesWithExtInfo ret     = new AtlasEntitiesWithExtInfo();
-        AlterDatabaseEvent       dbEvent = (AlterDatabaseEvent) context.getMetastoreEvent();
-        Database                 oldDb   = dbEvent.getOldDatabase();
-        Database                 newDb   = dbEvent.getNewDatabase();
-
-        if (newDb != null) {
-            AtlasEntity dbEntity = toDbEntity(newDb);
-
-            ret.addEntity(dbEntity);
-        } else {
-            LOG.error("AlterDatabase.getEntities(): failed to retrieve db");
-        }
-
-        addProcessedEntities(ret);
-
-        return ret;
-    }
-
-    public AtlasEntitiesWithExtInfo getHiveEntities() throws Exception {
-        return super.getHiveEntities();
-    }
-}
\ No newline at end of file
diff -Naur apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/BaseHiveEvent.java apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/BaseHiveEvent.java
--- apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/BaseHiveEvent.java	2019-05-06 12:55:33.000000000 -0500
+++ apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/BaseHiveEvent.java	2019-11-13 19:42:30.087556171 -0600
@@ -37,7 +37,6 @@
 import org.apache.hadoop.hive.metastore.api.Order;
 import org.apache.hadoop.hive.metastore.api.SerDeInfo;
 import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
-import org.apache.hadoop.hive.metastore.utils.SecurityUtils;
 import org.apache.hadoop.hive.ql.hooks.*;
 import org.apache.hadoop.hive.ql.hooks.LineageInfo.BaseColumnInfo;
 import org.apache.hadoop.hive.ql.hooks.LineageInfo.DependencyKey;
@@ -47,6 +46,7 @@
 import org.apache.hadoop.security.UserGroupInformation;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.apache.hadoop.hive.shims.Utils;
 
 import java.io.IOException;
 import java.net.URI;
@@ -689,7 +689,7 @@
 
         if (context.isMetastoreHook()) {
             try {
-                ugi = SecurityUtils.getUGI();
+                ugi = Utils.getUGI();
             } catch (Exception e) {
                 //do nothing
             }
diff -Naur apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveHook.java apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveHook.java
--- apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveHook.java	2019-05-06 12:55:33.000000000 -0500
+++ apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveHook.java	2019-11-13 19:40:12.087235438 -0600
@@ -174,12 +174,6 @@
                     event = new DropDatabase(context);
                 break;
 
-                case ALTERDATABASE:
-                case ALTERDATABASE_OWNER:
-                case ALTERDATABASE_LOCATION:
-                    event = new AlterDatabase(context);
-                break;
-
                 case CREATETABLE:
                     event = new CreateTable(context, true);
                 break;
diff -Naur apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveMetastoreHookImpl.java apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveMetastoreHookImpl.java
--- apache-atlas-sources-2.0.0/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveMetastoreHookImpl.java	2019-05-03 00:22:00.000000000 -0500
+++ apache-atlas-sources-2.0.0-patched/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/HiveMetastoreHookImpl.java	2019-11-13 19:41:21.239396244 -0600
@@ -24,7 +24,6 @@
 import org.apache.hadoop.hive.metastore.MetaStoreEventListener;
 import org.apache.hadoop.hive.metastore.api.FieldSchema;
 import org.apache.hadoop.hive.metastore.events.*;
-import org.apache.hadoop.hive.metastore.utils.SecurityUtils;
 import org.apache.hadoop.hive.ql.metadata.Table;
 import org.apache.hadoop.hive.ql.plan.HiveOperation;
 import org.apache.hadoop.hive.shims.Utils;
@@ -64,13 +63,6 @@
     }
 
     @Override
-    public void onAlterDatabase(AlterDatabaseEvent dbEvent) {
-        HiveOperationContext context = new HiveOperationContext(ALTERDATABASE, dbEvent);
-
-        hook.handleEvent(context);
-    }
-
-    @Override
     public void onCreateTable(CreateTableEvent tableEvent) {
         HiveOperationContext context = new HiveOperationContext(CREATETABLE, tableEvent);
 
@@ -126,10 +118,6 @@
                         event = new DropDatabase(context);
                         break;
 
-                    case ALTERDATABASE:
-                        event = new AlterDatabase(context);
-                        break;
-
                     case CREATETABLE:
                         event = new CreateTable(context, true);
                         break;
@@ -161,7 +149,7 @@
                 }
 
                 if (event != null) {
-                    final UserGroupInformation ugi = SecurityUtils.getUGI() == null ? Utils.getUGI() : SecurityUtils.getUGI();
+                    final UserGroupInformation ugi = Utils.getUGI();
 
                     super.notifyEntities(event.getNotificationMessages(), ugi);
                 }
@@ -190,4 +178,4 @@
 
         return isColumnRename;
     }
-}
\ No newline at end of file
+}
